tsconfig.json

```diff
@@ -24,7 +24,8 @@
         "suppressImplicitAnyIndexErrors": false,
         "target": "es2018",
         "declaration": false,
-        "sourceMap": false
+        "sourceMap": false,
+        "useUnknownInCatchVariables": false
```

Comment:
The updated typescript version started complaining about `error` being undefined when catching errors:
https://github.com/camshaft/install/blob/a780fd7a87fd99b3052579c99f46b38a46639bef/src/main.ts#L61

So I just ignored it.